bin/test-cleanup

```diff
@@ -3,26 +3,27 @@
 set -eu
 
 linkerd_namespace=${1:-l5d-integration}
+k8s_context=${2:-""}
 
 if [ -z "$linkerd_namespace" ]; then
```

Comment:
This will always be false as if first arg is not provided then linkerd-namespace will be set `l5d-integration` and if statement not executing.